src/Majordomo/MajordomoProtocolTests/MDPBrokerTests.cs

```diff
@@ -105,7 +105,7 @@ public async void Run_ReceiveREADYMessageFromWorker_LogSuccessfulRegistration ()
                 cts.Cancel ();
                 // check on the logging
                 Assert.That (log.Count, Is.EqualTo (2));
-                Assert.That (log[1], Is.StringContaining ("added to service echo"));
+                Assert.That (log[1], Does.Contain("added to service echo"));
```

Comment:
Why this change?

Reply:
Removed in Nunit 3.0
https://docs.nunit.org/articles/nunit/release-notes/breaking-changes.html